developer_manual/core/ocs-share-api.rst

```diff
@@ -129,3 +130,64 @@ Statuscodes:
 * 400 - wrong or no update parameter given
 * 403 - public upload disabled by the admin
 * 404 - couldn't update share
+
+
+Federated Cloud Shares
+======================
+
+Create a new Federated Cloud Share
+----------------------------------
+
+Creating a federated cloud share can be done via the local share endpoint, using
+(int) 6 as a shareType and the `Federated Cloud ID <https://owncloud.org/federation/>`_
+as shareWith. See `Create a new Share`_ for more information.
```

Comment:
You might want to make it clearer that this is the owner user sharing to the outside.